Source/Editor/Modules/ContentDatabaseModule.cs

```diff
@@ -1275,10 +1275,6 @@ public override void OnUpdate()
                 }
                 _dirtyNodes.Clear();
             }
-
-            // Lazy-rebuilds
-            if (_rebuildFlag)
```

Comment:
This still might be important for lazy-rebuilds - esp. if plugin adds multiple proxy types so it triggers only one rebuild on the next update.

Reply:
I forgot I removed this when I was doing some testing. I will add it back in.